x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/install_update_error_handling.cy.ts

```diff
@@ -233,7 +233,7 @@ describe(
         cy.get(UPGRADE_ALL_RULES_BUTTON).click();
         assertRuleUpgradeSuccessToastShown([OUTDATED_RULE_1]);
         assertRuleUpgradeFailureToastShown([OUTDATED_RULE_2]);
-        assertRulesPresentInRuleUpdatesTable([OUTDATED_RULE_1, OUTDATED_RULE_2]);
+        expectRulesInTable(RULES_UPDATES_TABLE, ['Outdated rule 1', 'Outdated rule 2']);
```

Comment:
I'm kind of curious why we test the rules are still here after asserting the toasts have thrown. Given the way the tests are setup, wouldn't this just be testing that the `cypress.intercept` call works? Even in this example where one rule "succeeds", they're both still present in the rules table

Reply:
Tbh I didn't touch these tests. This indeed looks weird.

Eventually I've decided to split the error handling tests for installation and upgrade workflows. On top of that I removed `expectRulesInTable(RULES_UPDATES_TABLE, ['Outdated rule 1', 'Outdated rule 2']);` from these tests to focus testing on toasts.